internal/config/channel.go

```diff
@@ -19,6 +20,7 @@ func (r *RuntimeConfig) fetchChannels(ctx context.Context, tx *sqlx.Tx) error {
 	}
 
 	channelsById := make(map[int64]*channel.Channel)
+	regEx, _ := regexp.Compile("^[a-zA-Z0-9]+$")
```

Comment:
If you don't care about the error and expect that the static regular expression MUST compile, use `regexp.MustCompile`. By doing so, you will realize that something went wrong and don't have to discard the error.

Reply:
Also, you might wanna give the variable a more meaningful name, e.g., `channelTypeNameRe` or the like.

Reply:
And move it outside of the function so that if someone would break the regex, they'd notice immediately when starting the program, not only when this code path is reached.